tests/cnf/core/network/sriov/tests/metricsExporter.go

```diff
@@ -488,3 +506,19 @@ func setMetricsExporter(flag bool) {
 	_, err = defaultOperatorConfig.Update()
 	Expect(err).ToNot(HaveOccurred(), "Failed to update metricsExporter flag in default Sriov Operator Config")
 }
+
+func clearClientServerMacTableFromSwitch() {
+	switchCredentials, err := sriovenv.NewSwitchCredentials()
+	Expect(err).ToNot(HaveOccurred(), "Failed to get switch credentials")
+
+	jnpr, err := cmd.NewSession(switchCredentials.SwitchIP, switchCredentials.User, switchCredentials.Password)
+	Expect(err).ToNot(HaveOccurred())
```

Comment:
Please descriptions to all Expects in this func

Reply:
done